deep-learning/deep-learning-index.html

```diff
@@ -5,6 +5,7 @@
 background-class: deep-learning-background
 permalink: deep-learning-with-pytorch
 deep-learning-landing: true
+deep-learning-error: false
```

Comment:
Do you plan on using these `deep-learning-error` variables? It doesn't look like they're being used anywhere in this PR.

Reply:
I don't think I plan on using it anymore. I initially kept it because I thought it might be needed for the back-end work.